src/controllers/manifests.js

```diff
@@ -33,9 +33,9 @@ exports.create = function (client, storage, pwabuilder, raygun) {
           .fail(function (err) {
             return res.status(422).json({ error: err.message });
           });
-      } else if (req.body.manifest) {
+      } else if (req.body.content && req.body.format === "w3c") {
         pwabuilder
-          .uploadManifest(req.body.manifest, client)
+          .uploadManifest(req.body, client)
```

Comment:
Do we need to update `uploadManifest` to handle the manifest properly? The UI is sending in a ManifestDetectionResult object, which contains a `.content` property with the actual manifest. Does `uploadManifest` need to handle that?

Reply:
And to respond to your comment, the format saved in redis is the format that has the actual manifest is the one we're passing e.g.

{
  content: ManifestDetectionResult,
  default...
  ad naseum...
  id: "crypto has uuid etc"
}